framework/src/test/java/org/tron/core/services/http/UtilTest.java

```diff
@@ -108,4 +130,42 @@ public void testPackTransactionWithInvalidType() {
     Assert.assertEquals("Invalid transaction: no valid contract",
         txSignWeight.getResult().getMessage());
   }
+
+  @Test
+  public void testPackTransaction() {
```

Comment:
Please submit a new PR if necessary

Reply:
It is used to improve coverage.

Reply:
But it doesn't seem to have anything to do with stake code 2.0, if the coverage decreases, try to increase the patch coverage, https://github.com/tronprotocol/java-tron/issues/5438.

Reply:
I removed it.